release/build.sh

```diff
@@ -9,7 +9,7 @@ build () {
   exeName="$4"
   echo "Building $exeName for $GOOS-$GOARCH ..."
 
-  res=CGO_ENABLED=0 go build -o "$exeName" -ldflags '-w -extldflags "-static"' main.go
+  res=CGO_ENABLED=0 jf go build -o "$exeName" -ldflags '-w -extldflags "-static"' main.go
```

Comment:
I suggest we also grant the ```bi``` executable execute permission, so that people don't have to run ```chmod +x``` after they download it.

Reply:
Good point! 🤔